nava/thread.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+"""Nava thread."""
+
+import sys
+import threading
+
+
+class NavaThread(threading.Thread):
+    """Nava custom thread."""
+
+    def __init__(self, *args, **kwargs):
+        """
+        Init method.
+
+        :param args: arguments
+        :type args: list
+        :param kwargs: keyword arguments
+        :type kwargs: dict
+        """
+        super(NavaThread, self).__init__(*args, **kwargs)
+        self.play_process = None
+
+    def run(self):
+        """
+        Run target function.
+
+        :return: None
+        """
+        if self._target is not None:
+            self.play_process = self._target(*self._args, **self._kwargs)
+
+    def stop(self):
+        """
+        Stop sound.
+
+        :return: None
+        """
+        sys_platform = sys.platform
+        if sys_platform == "win32":
+            import winsound
+            winsound.PlaySound(None, winsound.SND_PURGE)
```

Comment:
Did you test the case in Windows where you have multiple sounds playing asynchronously and try to stop just one of them?

Reply:
Currently, the `winsound` module cannot play multiple sounds simultaneously. But play/stop of one sound in async mode works like charm!